test/language/expressions/tagged-template/call-expression-context.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) Copyright 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.3.7
+description: Invocation context for call expressions
+info: >
+    A tagged template is a function call where the arguments of the call are
+    derived from a TemplateLiteral. The actual arguments include a template
+    object and the values produced by evaluating the expressions embedded
+    within the TemplateLiteral.
+---*/
+var context;
+var fn = function() {
+  return function() {
```

Comment:
Good catch! This is a bug in the test as indicated by the file's name, "call-expression-context.js". This ought to be asserting that a tagged template applied to a call expression has the correct context. That means the frontmatter needs the `onlyStrict` flag and the invocation should read `fn()LITERALBACKTICKCHARACTERHERENoSubstitutionTemplateLITERALBACKTICKCHARACTERHERE`

(I need to find a better convention for using backticks within inline code samples)